app/RDF/RdfServiceProvider.php

```diff
@@ -41,10 +40,6 @@ public function register(): void
         $this->container->addShared(
             RdfResourceFactory::class,
             function (): RdfResourceFactory {
-                if ($this->getContainer()->get('config')['rdf']['blank_nodes_allowed'] ?? true) {
-                    return new RdfResourceFactoryWithBlankNodes();
-                }
-
                 return new RdfResourceFactoryWithoutBlankNodes(new NodeUriGenerator(new CRC32HashGenerator()));
```

Comment:
Is it still relevant to keep the name `RdfResourceFactoryWithoutBlankNodes`, because `RdfResourceFactoryWithBlankNodes` no longer exist. I think the new class should be renamed to `RdfResourceFactory`

Reply:
Good point, I removed the interface and renamed the class.